dashboard/app/controllers/pd/misc_survey_controller.rb

```diff
@@ -36,6 +36,8 @@ def new
         submitRedirect: url_for(action: 'submit', params: {key: key_params})
       )
 
+      return if experimental_redirect! @form_id, @form_params
```

Comment:
We could maybe just wrap the below code in an `unless` to make it clearer that we're explicitly skipping the NewRelic work because we're in the experiment.

Reply:
Ditto for all the other files too.

Reply:
Though, I get that we're doing work in this statement... so maybe it's okay since it's temporary.